packages/cyberstorm/src/components/Layout/LoginLayout/BetaLoginLayout.tsx

```diff
@@ -1,14 +1,20 @@
 import styles from "./BetaLoginLayout.module.css";
-import { Button } from "../../Button/Button";
+import { Button, PlainButton } from "../../Button/Button";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faDiscord, faGithub } from "@fortawesome/free-brands-svg-icons";
 import { OverwolfLogo, ThunderstoreLogo } from "../../../svg/svg";
 import { PrivacyPolicyLink, TermsOfServiceLink } from "../../Links/Links";
 
+export interface BetaLoginLayoutProps {
```

Comment:
I'm not sure this needs to be exported. Also for brevity it could be just `Props`, since the context makes it clear what it's used for.

Reply:
It does not need to be exported 👍 Though I'm not sure about the naming thing, I it when those are "specifically named", but I don't mind going by simpler names 🤷

Reply:
I'm not sure about best practices either, but ideally we have one component per file, so "Props" IMO says enough without causing confusion. Things change a bit if the props are exported since then multiple "Props" can be imported in a single file (although this can be resolved with "import x as y").

This comment can be marked as resolved.

Reply:
Agreed 💯